ci/package.sh

```diff
@@ -127,7 +124,7 @@ do
                             # force build of template archive if it doesn't exist
                             build=true
                         fi
-
+                    
```

Comment:
Would be nice to get rid of this whitespace

Reply:
Have removed and pushed new changes